contracts/0.8.25/vaults/Dashboard.sol

```diff
@@ -194,13 +277,33 @@ contract Dashboard is AccessControlEnumerable {
     }
 
     /**
-     * @notice Burns stETH tokens from the sender backed by the vault
+     * @notice Mints wstETH tokens backed by the vault to a recipient. Approvals for the passed amounts should be done before.
+     * @param _recipient Address of the recipient
+     * @param _tokens Amount of tokens to mint
+     */
+    function mintWstETH(address _recipient, uint256 _tokens) external payable virtual onlyRole(DEFAULT_ADMIN_ROLE) fundAndProceed {
+        _mint(_recipient, _tokens);
+        IWstETH(wstETH).wrap(_tokens);
+    }
+
+    /**
+     * @notice Burns stETH tokens from the sender backed by the vault. Approvals for the passed amounts should be done before.
      * @param _tokens Amount of tokens to burn
      */
     function burn(uint256 _tokens) external virtual onlyRole(DEFAULT_ADMIN_ROLE) {
         _burn(_tokens);
     }
 
+    /**
+     * @notice Burns stETH tokens from the sender backed by the vault using EIP-2612 Permit.
+     * @param _tokens Amount of tokens to burn
+     * @param _permit data required for the stETH.permit() method to set the allowance
+     */
+    function burnWithPermit(uint256 _tokens, PermitInput calldata _permit) external virtual onlyRole(DEFAULT_ADMIN_ROLE) {
```

Comment:
burnWsteth(WithPermit)

Reply:
I wasn't sure that burning wstETH works on the business logic of Vaults. But let's add it and check on the review

Reply:
like on WQ, you transfer wsteth to this contract balance, unwrap to steth and procced with burn

Reply:
Yes. I meant the Vault economy which takes into account the burning of tokens. However, I have already added